Cargo.toml

```diff
@@ -26,6 +26,7 @@ force_enable_x86_64_darwin_generations = []
 target_postgrestd = []
 
 [dependencies]
+current_platform = "0.2.0"
```

Comment:
This dependency holds less actual code than the `leftpad` of npm fame did.
```suggestion
```

Reply:
hahaha that's fair